public/javascripts/payments.js

```diff
@@ -573,6 +573,17 @@
     }
   };
 
+  /**
+   * Check if the PaymentIntent is in a "terminal" status
+   * and therefore if we should show an error in the UI
+   */
+  const paymentIntentTerminal = ({status, last_payment_error}) => {
+    const endStates = ['succeeded', 'processing', 'canceled'],
+          hasError = typeof last_payment_error !== "undefined";
```

Comment:
I couldn't find any other instances of `,` to instantiate multiple `const`s in this codebase. Should we just `const` twice?